docs/source/overview/cache_groups.rst

```diff
@@ -31,4 +49,273 @@ All servers have to have a :term:`Physical Location`, which defines their geogra
+
+.. seealso:: :ref:`health-proto`
+
+.. table:: Aliases
+
+	+--------------------------+----------------------+----------------------------------------+
+	| Name                     | Use(s)               | Type(s)                                |
+	+==========================+======================+========================================+
+	| Fallback to Geo Failover | Traffic Portal forms | Unchanged (``bool``, ``Boolean`` etc.) |
+	+--------------------------+----------------------+----------------------------------------+
+
+.. _cache-group-id:
+
+ID
+--
+All Cache Groups have an integral, unique identifier that is mainly used to reference it in the :ref:`to-api`.
+
+Despite that a Cache Group's Name_ must be unique, this is the identifier most commonly used to represent a unique Cache Group in most contexts throughout :abbr:`ATC (Apache Traffic Control)`. One notable exception is in CDN :term:`Snapshots` and in routing configuration used by Traffic Router.
+
+.. _cache-group-latitude:
+
+Latitude
+--------
+The Cache Group's geomagnetic latitude for use in routing and for the purposes of `Fallback to Closest`_.
+
+.. table:: Aliases
+
+	+-----------------------+----------------------+----------------------------------------+
+	| Name                  | Use(s)               | Type(s)                                |
+	+=======================+======================+========================================+
+	| Geo Magnetic Latitude | Traffic Portal forms | Unchanged (``number``, ``float`` etc.) |
+	+-----------------------+----------------------+----------------------------------------+
+
+.. _cache-group-localization-methods:
+
+Localization Methods
+--------------------
+The :dfn:`Localization Methods` of a Cache Group define the methods by which Traffic Router is allowed to route clients to :term:`cache servers` within this Cache Group. This is a collection of the allowed methods, and the values in the collection are restricted to the following.
+
+- "Coverage Zone File" (alias ``CZ`` in source code, database entries, and :ref:`to-api` requests/responses) allows Traffic Router to direct clients to this Cache Group if they were assigned a geographic location by looking up their IP address in the :term:`Coverage Zone File`.
+- "Deep Coverage Zone File" (alias ``DEEP_CZ`` in source code, database entries, and :ref:`to-api` requests/responses) allows Traffic Router to direct clients to this Cache Group if they were assigned a geographic location by looking up their IP addresses in the :term:`Deep Coverage Zone File`.
+
+	.. warning:: In order to make use of "deep caching" for a :term:`Delivery Service`, this localization method must be enabled on the Cache Groups that contain servers assigned to said :term:`Delivery Service` **and** that :term:`Delivery Service` must have :ref:`ds-deep-caching` enabled.
```

Comment:
I don't think this is really true. For example, let's say cache1 is in cachegroup1. Cachegroup1 only has CZ and GEO enabled. Let's say cache1 is in a "deep" cachegroup per the DCZF. Cache1 will still be routed to as part of the "deep" cachegroup, even though cache1's "real" cachegroup -- cachegroup1 -- doesn't have DEEP_CZ enabled. This is because "deep" cachegroups do not map to "real" cachegroups in TO -- they are a concept only within the DCZF.

Basically, I think this warning, along with the "deep coverage zone file" bullet point, can be removed entirely. It can be enabled for a TO cachegroup, but it doesn't actually change/do anything for that cachegroup.

Reply:
How do you feel about what I just pushed up?

Reply:
better